package.json

```diff
@@ -170,18 +170,18 @@
     "@gnosis.pm/safe-contracts": "1.1.1-dev.2",
     "@gnosis.pm/safe-react-components": "https://github.com/gnosis/safe-react-components.git#03ff672d6f73366297986d58631f9582fe2ed4a3",
     "@gnosis.pm/util-contracts": "2.0.6",
-    "@ledgerhq/hw-transport-node-hid-singleton": "5.29.0",
+    "@ledgerhq/hw-transport-node-hid-singleton": "5.30.0",
     "@material-ui/core": "4.11.0",
     "@material-ui/icons": "4.9.1",
     "@material-ui/lab": "4.0.0-alpha.56",
     "@openzeppelin/contracts": "3.1.0",
-    "@sentry/react": "^5.27.3",
-    "@sentry/tracing": "^5.27.3",
-    "@truffle/contract": "4.2.28",
+    "@sentry/react": "^5.27.0",
```

Comment:
why the decreased version?

Reply:
hm!... nevermind, the version will be correct after all.

Reply:
Yes, it was a test on handling dependencies in a more flexible way. This allows versions `< 5.27.0` anyway and yarn resolve it like the latest `5.27.x`